README-ja.md

```diff
@@ -1,16 +1,18 @@
 <div align="right">
 
-  [Readme in English](README.md) | [中文](README-zh.md) | [正體中文](README-zh-TW.md)
+  [中文](README-zh.md) | [正體中文](README-zh-TW.md) | [日本語で読む](README-ja.md)
```

Comment:
This is a navbar to link to other languages, so it is definitely not in the same order or using the same text in this area.